_includes/tutorials/fk-joins/kstreams/markup/test/make-test-file.adoc

```diff
@@ -0,0 +1,8 @@
+First, create a test file at `configuration/test.properties`:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/fk-joins/kstreams/code/configuration/test.properties %}</code></pre>
++++++
+
+You should take note of the `schema.registry.url` configuration.  The config is using a special pseudo-protocol `mock://..` which means our test
```

Comment:
just added this bit text as it explains the test configuration for schema registry i.e. using `MockSchemaRegistry`


\cc @colinhicks @rspurgeon